config/Dockerfiles/release-notes.py

```diff
@@ -129,5 +129,9 @@ def format_body(tasks):
 release_body = format_body(tasks)
 release_name = milestone
 
-repo.create_git_release(milestone, release_name, release_body,
-                        target_commitish="master")
+release = repo.create_git_release(milestone, release_name, release_body,
```

Comment:
@abraverm I am not familiar with appImage, but looking at the script part of the yaml, it looks good to me. If we don't want to risk it we could either maintain different/seperate Docker Image just for appImage release initially. 

The release pipeline itself can be tested locally in contra-env-setup with the current Jenkinsfile if I see it right.